src/pages/index.ejs

```diff
@@ -203,7 +203,9 @@ description: "Betaflight is flight controller software (firmware) used to fly mu
 							<div class="container">
 								<div class="video-content">
 			
-									<iframe width="560" height="315"  src="https://www.youtube.com/embed/videoseries?list=PLwoDb7WF6c8nT4jjsE4VENEmwu9x8zDiE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+									<div class="video-content">
```

Comment:
This div and class are already defined and opened

Reply:
The previous rev with the 4.2 videos used this presentation as it was a single video. When I put bardwells videos in they are a playlist which can only be embedded though an iframe. This help video is now back to a single video which I felt suited the oringal styling. My sense is either one works usability wise and I can go either way.

Reply:
Good! But please remove this extra container, it already exist